package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Optimistic Nightfall",
   "main": "./nightfall-client/src/index.mjs",
   "scripts": {
+    "preinstall": "npx npm-force-resolutions",
```

Comment:
Did you try to update the top level `colors` package before doing this?

Reply:
you're right, this shouldn't be necessary.
I settled for pinning the dependency version using overrides.
Unfortunately, the GH dependency check still fails, even if a non vulnerable version is pinned.